src/custom/components/swap/ConfirmSwapModal/index.tsx

```diff
@@ -1,4 +1,32 @@
-import ConfirmSwapModal from './ConfirmSwapModalMod'
+import React from 'react'
+import { Trans } from '@lingui/macro'
+
+import ConfirmSwapModalMod from './ConfirmSwapModalMod'
+import TradeGp from 'state/swap/TradeGp'
+import { formatSmart } from 'utils/format'
 
 export * from './ConfirmSwapModalMod'
-export default ConfirmSwapModal
+
+function buildPendingText(trade: TradeGp | undefined): JSX.Element {
```

Comment:
nit: should be capitalised if it's a component, probably can just be `PendingText`

Reply:
Good point, will update